navpy/core/navpy.py

```diff
@@ -1121,9 +1127,12 @@ def ecef2ned(ecef,lat_ref,lon_ref,alt_ref,latlon_unit='deg',alt_unit='m',model='
     
     Examples
     --------
-    >>> import numpy as np
-    >>> from navpy import ecef2ned
-    >>> lat 
+    >>> import navpy
```

Comment:
The same comments about style/format.  

The example variables/description appears to be suitable for `lla2ned`.  It would need ECEF positions to be suitable for the `ecef2ned` function.  If further clarification is needed please say so.

Reply:
Looks like proper usage. Again referencing the numpydoc style guide, can you keep lines to < 75 characters?  Also add the last print output.
> The length of docstring lines should be kept to 75 characters to facilitate reading the docstrings in text terminals.